clouddrift/sphere.py

```diff
@@ -435,3 +435,133 @@ def sphere_to_plane(
     y[..., :] = np.cumsum(dy, axis=-1)
 
     return x, y
+
+
+def earthsphere_to_cartesian(
```

Comment:
Why not `sphere_to_cartesian`? Unless the ellipsoid stuff will be Earth-specific. But it seems to me that these functions will be useful on other planets too, e.g. Mars.

Reply:
ok!